src/components/DataTable/defaultSelectAllRenderer.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import cx from 'classnames';
 
 import { CheckboxRaw } from '../../';
 
-export default ({ allSelected, dataKey, onSelectAll, tableId }) => { // eslint-disable-line react/prop-types
+const FIXED_STYLE = { marginLeft: '-0.25rem' };
+
+export default ({ allSelected, dataKey, fixed, onSelectAll, tableId, truncate }) => { // eslint-disable-line react/prop-types
   const checkboxId = `${tableId}-${dataKey}`;
+  const wrapperClassName = cx([
+    { 'slds-truncate': truncate },
```

Comment:
What is being truncated here? As far as I am aware, this only renders a checkbox and no text